packages/@uppy/utils/src/delay.test.js

```diff
@@ -30,7 +30,7 @@ describe('delay', () => {
 
     // should have rejected before the timer is done
     const time = Date.now() - start
-    expect(time).toBeGreaterThanOrEqual(50)
-    expect(time).toBeLessThan(100)
+    expect(time).toBeGreaterThanOrEqual(30)
+    expect(time).toBeLessThan(70)
```

Comment:
tbh using `Date` API is probably not a great idea for this kind of tests, it could use a refactor to `performance.now()`. But a change for another time, it should not block this PR.